Ruby/Ruby.sublime-syntax

```diff
@@ -1030,8 +1030,72 @@ contexts:
           pop: true
         - include: interpolated-ruby
         - include: escaped-char
+    # literal capable of interpolation ()
+    - match: '%[I]?\('
```

Comment:
An above rule matching `[QWS]?` will match before `I?`, so the `?` isn't useful. The square brackets aren't necessary since there aren't multiple letters that should be possible options.

```suggestion
    - match: '%I\('
```